client/src/api/locations.js

```diff
@@ -1,25 +1,19 @@
-export async function getLocationNamebyTour() {
-  const response = await fetch(`/api/locations/electronic`);
-
+export async function getLocationByName(name) {
+  const response = await fetch(`/api/locations/name/${name}`);
```

Comment:
According to another comment further down, if you would want to adapt that, you would need to change it here as well.
```suggestion
export async function getLocationByName(name) {
  const response = await fetch(`/api/locations?name=${name}`);
```

Reply:
Implemented 🙌 THX for suggesting this.